components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -2801,18 +2710,10 @@ export function createWalletApi (
             const { cache, data: api } = baseQuery(undefined)
             const { ethTxManagerProxy } = api
 
+            // TODO(apaymyshev): this should not be selected account and network,
+            // but account and network for watched transaction
```

Comment:
Let's attach a follow-up issue for this. Will probably need to add args to the query so it knows which account/network to check